blockchain/chain_sync/src/sync_worker.rs

```diff
@@ -283,6 +283,8 @@ where
     /// fork detected, collect tipsets to be included in return_set sync_headers_reverse
     async fn sync_fork(&self, head: &Tipset, to: &Tipset) -> Result<Vec<Arc<Tipset>>, Error> {
         // TODO move to shared parameter (from actors crate most likely)
+        // TODO the threshold should really be 900. Not entirely sure if we can handle a 900
```

Comment:
The todo a few lines below is the same, maybe just delete that old one or combine somehow to remove confusion?